ClimsoftVer4/ClimsoftVer4/formSynopRA1.vb

```diff
@@ -14,7 +14,6 @@
 ' You should have received a copy of the GNU General Public License
 ' along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-
```

Comment:
In this file there were two blank lines between the licence info and the first line of code and GitHub shows me that you removed one of these blank lines (which is a good thing to do - let's have either one blank line or no blank lines here)